app/routes.py

```diff
@@ -1 +1,205 @@
-from flask import Blueprint
\ No newline at end of file
+from flask import Blueprint, jsonify, abort, make_response, request
+from app import db
+from app.models.task import Task
+from app.models.goal import Goal
+from datetime import date
+import requests
+import os
+
+
+def validate_model(cls, model_id):
+    model = cls.query.get(model_id)
+   
+    if not model:
+        abort(make_response({"message":f"{cls.__name__} {model_id} not found"}, 404))
+    
+    return model
+
+# ************************* CRUD ROUTES FOR TASKS ****************************************
+
+tasks_bp = Blueprint("tasks", __name__, url_prefix="/tasks")
+
+@tasks_bp.route("", methods=["POST"])
+def create_task():
+    request_body = request.get_json()
+
+    if request_body.get("title") and request_body.get("description"):
+        new_task = Task.from_json(request_body)
+    else:
+        return jsonify({"details": "Invalid data"}), 400
```

Comment:
Nice work, but I would provide the user with information about what data is invalid.